firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/internal/InstallActivity.java

```diff
@@ -32,8 +33,14 @@
  */
 public class InstallActivity extends AppCompatActivity {
   private static final String TAG = "InstallActivity: ";
-  private boolean unknownSourceEnablementInProgress = false;
   private boolean installInProgress = false;
+  private AlertDialog alertDialog;
```

Comment:
Can you rename this variable to `enableUnknownSourcesDialog` or something? It would help me understand the logic behind the `if` statement in onResume, for example.